X-Science/StatusWindow.cs

```diff
@@ -323,7 +323,7 @@ public void UpdateSituation( object sender, NewSituationData e )
 			_logger.Trace( "ScienceThisBiome: " + _filter.TotalCount + " / " + _filter.CompleteCount );
 			if( _filter.TotalCount > 0 )
 			{
-				if( _filter.TotalCount - _filter.CompleteCount > 0 )
+				if( _filter.DisplayScienceInstances.Any(x => CanRunExperiment(x)) )
```

Comment:
I have created a new commit that replaces linq with a regular loop. I could not create a pull request this time. I just don't really know how it works. I hope the new commit is visible to you. If not, contact me on the forums.